cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2-datascience.json

```diff
@@ -0,0 +1,2869 @@
+[
+  {
+    "abstract": {
+      "description": "<p>Simulated dataset BulkGravTohhTohbbhbb_narrow_M-600_13TeV-madgraph in <a href=\"https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookMiniAOD2016\">MINIAODSIM</a> format for 2016 collision data.</p><p>See the description of the simulated dataset names in: <a href=\"/about/CMS-Simulated-Dataset-Names\">About CMS simulated dataset names</a>.</p> <p>These simulated datasets correspond to the collision data collected by the CMS experiment in 2016.</p> <p>They were released in the context of data science sample production.</p>"
+    },
+    "accelerator": "CERN-LHC",
+    "categories": {
+      "primary": "Exotica",
+      "secondary": [
+        "Gravitons"
+      ],
+      "source": "CMS Collaboration"
+    },
+    "collaboration": {
+      "name": "CMS Collaboration",
+      "recid": ""
```

Comment:
Please remove empty recid for now, and create an issue to add a new CMS author file record for 2016.
(`cms-author-list-2016.json`) CC @ArtemisLav @katilp